File: BackendMoura/Models/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BackendMoura.Models
{
    public class Pessoa
    {
        public int Codigo { get; set; }
        public string Nome { get; set; }
        public DateTime DataNascimento { get; set; }
        public bool Inativo { get; set; }
        public NacionalidadeEnum Nacionalidade { get; set; }
        public string RG { get; set; }
        public string Passaporte { get; set; }
    }

    public enum NacionalidadeEnum
    {
        BR = 1,
        Outros = 2
    }

}

File: FrontendMoura/src/app/Application/Classes/Pessoa.ts
import { IPessoa } from "../Interfaces/IPessoa";

export class Pessoa implements IPessoa {
  Codigo: number;
  Nome: string;
  DataNascimento: Date;
  Inativo: boolean;
  Nacionalidade: number;
  RG: string;
  Passaporte: string;

  constructor(
    Codigo: number,
    Nome: string,
    DataNascimento: Date,
    Inativo: boolean,
    Nacionalidade: number,
    RG: string,
    Passaporte: string
  ) {
    this.Codigo = Codigo;
    this.Nome = Nome;
    this.DataNascimento = DataNascimento;
    this.Inativo = Inativo;
    this.Nacionalidade = Nacionalidade;
    this.RG = RG;
    this.Passaporte = Passaporte;
  }
}
